Exiled.Events/EventArgs/Player/DroppingAmmoEventArgs.cs

```diff
@@ -52,7 +52,7 @@ public DroppingAmmoEventArgs(Player player, AmmoType ammoType, ushort amount, bo
         /// <summary>
         ///     Gets or sets the amount of ammo being dropped.
         /// </summary>
-        public int Amount { get; set; }
```

Comment:
Use Obsolete instead of removing it and cast it to Ushort ?

Reply:
`ushort` is what the game uses, how is he going to cast it?